components/fxa-client/src/ffi.rs

```diff
@@ -125,6 +125,7 @@ pub struct ProfileC {
     uid: *mut c_char,
     email: *mut c_char,
     avatar: *mut c_char,
+    avatar_default: bool,
```

Comment:
👍

Reply:
Don't use bool for things exposed to JNA, it has a bug where it thinks bools are 32 bits.

Reply:
gosh, not even booleans are safe :((((